db.json

```diff
@@ -502,20 +523,26 @@
     "source": "iana"
   },
   "application/font-sfnt": {
-    "source": "iana"
+    "source": "iana",
+    "extensions": ["otf","ttf"]
```

Comment:
This mapping is obsolete; many of these font types (moved under font/ tree. More information can be found here: http://tools.ietf.org/rfc/rfc8081.txt

Reply:
Obsolete yes. Is not being obsolete prerequisite for the extension inclusion? Many files are going to have that extension regardless of whether someone declares is obsolete.

Reply:
The mime is what is obsolete, not the extension. The files would simply use the non-obsolete file extension. My comment is about mapping the extension here to the obsolete type instead of the current type.